src/main/java/com/ericsson/ei/handlers/ObjectHandler.java

```diff
@@ -274,6 +275,7 @@ public String lockDocument(String id) {
      *
      * @return ttl Integer value representing time to live for documents
      */
+    @JsonIgnore
```

Comment:
Any special reason we add them here? No changes in code any way.
Can we remove them.

Reply:
Check EIInstanceInfo... class,, that one parses all Handlers objects with Jackson Json library and all getter(+ lombok getter) functions to a summarized Json obejct that is return back ,"/information" entrypoint , to EI Frontend for the EI Info page.
JsonIgnore just tell to skip this getter function, since there is already a lombok getter for "arggregationTtl" in top of this file.

Reply:
ok

Reply:
Same small bug exist in EI 2.x, so someone can update with same change in EI 2.x maintenance branch.